File: micro-business/src/main/java/com/jiangtj/micro/business/config/SystemConfigProperties.java
package com.jiangtj.micro.business.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

import java.util.HashMap;
import java.util.Map;

@Data
@ConfigurationProperties(prefix = "system.config")
public class SystemConfigProperties {
    private Boolean enabled = false;
    private String filePath = "upload";
    private Map<String, String> kv = new HashMap<>();
}

File: micro-business/src/main/java/com/jiangtj/micro/business/config/SystemConfigProperties.kt
package com.jiangtj.micro.business.config

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "system.config")
data class SystemConfigProperties(
    var enabled: Boolean = false,
    var filePath: String = "upload",
    var kv: MutableMap<String, String> = mutableMapOf(),
)
